### PR TITLE
Completed react-validated-input-component exercise. - Daniel Cho

### PR DESCRIPTION
**Pull request by Daniel-J-Cho**
Kapture 2022-05-20 at 16 55 05

### CODE REVIEW

File: react-validated-input-component/src/validated-input.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+class ValidatedInput extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { password: '', errorMsg: '' };
```
Comment (TimelikeClosure) [inline react-validated-input-component/src/validated-input.jsx]: The `errorMsg` state doesn't look like it's being used, so I'd suggest getting rid of it.

File: react-validated-input-component/src/validated-input.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    } else if (this.state.password.length < 8) {
+      errMsg = <p>Your password is too short.</p>;
+      icon = <i className="fa-solid fa-xmark"></i>;
+    } else {
+      icon = <i className="fa-solid fa-check"></i>;
+    }
+    return (
+      <div className="container">
+        <p className="password">Password</p>
+        <input type="password" value={this.state.password} onChange={this.handlePasswordChange}></input>
```
Comment (TimelikeClosure) [inline react-validated-input-component/src/validated-input.jsx]: Use labels for form controls instead of paragraphs.

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

File: react-validated-input-component/src/validated-input.jsx (hunk 1 above)
Comment (Daniel-J-Cho) [reply react-validated-input-component/src/validated-input.jsx]: Noted and corrected

File: react-validated-input-component/src/validated-input.jsx (hunk 2 above)
Comment (Daniel-J-Cho) [reply react-validated-input-component/src/validated-input.jsx]: Noted and corrected

**Review by scbowler — APPROVED** [review]
Looks good! 👍

**Pull request merged**